hexa/ui/templates/ui/icons/eye.html

```diff
@@ -0,0 +1,5 @@
+<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" class="w-6 h-6">
```

Comment:
Can you check how to make them customisable with variables please.

Reply:
Okay.